onnxruntime/core/platform/posix/env.cc

```diff
@@ -355,7 +355,7 @@ class PosixEnv : public Env {
   }
 
   std::string GetRuntimePath() const override {
```

Comment:
should you just remove this override method?
it already has default implementation which matches this.